agent/config/parse_linux.go

```diff
@@ -45,6 +45,14 @@ func parseGMSACapability() bool {
 					return false
 				}
 			}
+
+			//skip domain join check if the domainless gMSA is supported
+			domainLessgMSAUser := os.Getenv("CREDENTIALS_FETCHER_SECRET_NAME_FOR_DOMAINLESS_GMSA")
```

Comment:
- is this a new env var? If so can we add it to the README? is this env var a bool?
- can the env var name be shortened to something like `CREDENTIALS_FETCHER_DOMAINLESS_GMSA`?
- nit: `domainlessGMSAUser`

Reply:
will add to readme